sql/1_2022_12_14_add_title_h_location.sql

```diff
@@ -0,0 +1,4 @@
+\c grottoce 
```

Comment:
As this is an initial error in the model (and this table is still unused in production) could you please change the 0_tables.sql and 0_triggers.sql files instead (and delete these 2 files) ?

Can you just provide these 2 below queries (+ the missing query to fix the trigger) as Slack message to apply it manually on the prod database ?